Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla40251.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+// Apply the default category of "Issues" to all of the tests in this assembly
+// We use this as a catch-all for tests which haven't been individually categorized
+#if UITEST
+[assembly: NUnit.Framework.Category("Issues")]
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Bugzilla, 40251, "Cannot style Buttons natively using UIButton.Appearance", PlatformAffected.iOS)]
+	public class Bugzilla40251 : TestContentPage // or TestMasterDetailPage, etc ...
+	{
+		protected override void Init()
+		{
+			Content = new ContentView
+			{
+				Content = new Button
+				{
+					HorizontalOptions = LayoutOptions.Center,
+					VerticalOptions = LayoutOptions.Center,
+					Text = "Click",
+					BackgroundColor = Color.Black,
+					WidthRequest = 250,
+					HeightRequest = 50
+				}
+			};
+		}
+	}
+}
```

Comment:
Since this isn't an automated test, please add some instructions for the reviewer so they can determine if it's passing or not. Also, this doesn't really seem to test the changed behavior; could this test be modified to update the proxy appearance and then create a new button? That way we're demonstrating what the change does and providing a code example.